jenkinsfile.groovy

```diff
@@ -694,7 +694,7 @@ pipeline {
                 script: "npm run build:${databaseTestLoaderModuleName}",
                 returnStatus: true
             )) {
-              failure("Failed to install database loader script.")
+              failure("Failed to install database loader script. Check the logs above for vulnerable dependencies.")
```

Comment:
It suggests that fail occurred because of vulnerable dependencies.

Reply:
I've amended the message to make it clear that vulnerabilities may be one of the possible causes.